### PR TITLE
MarkDown formatting fixes

### PR DESCRIPTION
**Pull request by markhu**
Inspired by #108 / #109 Readme updates.

MD ideas from  https://stackoverflow.com/questions/6046263/how-to-indent-a-few-lines-in-markdown-markup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,13 +150,12 @@ pairs as shown below:
...
 
-  5. Zip up this file by typing the following into the command line. The command consists of the first filename which is the zip file you want to create _(call it whatever you like .zip)_ followed by the files you want to zip up. In our example you can see the name of the ```.js``` file we created earlier:
-
-   `$ zip -r hello-world.zip hello-world.js`
-
-    You should now be able to see a ```.ZIP``` file alongside your ```.js``` file.  
-    **NOTE: If your function has any dependencies then you must include your ```node_modules``` file within your .ZIP file. Simply add ```node_modules``` after the files you wish to zip up!**
+5. Zip up this file by typing the following into the command line. The command consists of the first filename which is the zip file you want to create _(call it whatever you like .zip)_ followed by the files you want to zip up. In our example you can see the name of the ```.js``` file we created earlier:
+   
+ &nbsp; &nbsp; &nbsp; &nbsp; `$ zip -r hello-world.zip hello-world.js`
```
Comment (markhu) [inline README.md]: There is a better way, using child-indicating 4-space-prefixed lines. The trick is, you have to enclose code with triple-backticks in multi-line format.

File: README.md [hunk 2]
```diff
@@ -603,7 +603,7 @@ Amazon SNS is a Publisher/Subscribe System. You can create, subscribe and publis
 
   ![create topic pop up](https://cloud.githubusercontent.com/assets/5912647/12579906/54c371de-c423-11e5-8982-9668384cf90a.png)
 
-#### Create a Lambda Function and Subscribe to a topic
+#### Create a Lambda Function and Subscribe to a topic
```
Comment (markhu) [inline README.md:607]: Unsure what kind of strange invisible space we originally used here, but it broke the MarkDown.

File: README.md [hunk 3]
```diff
@@ -1079,7 +1079,7 @@ We will be writing our own bash script that will involve the use of some of the
 
   [Gulp](https://github.com/gulpjs/gulp/blob/master/docs/API.md) can be used to automate the zipping, deployment and testing of Lambda functions on AWS. The Codeship deployment script can then be reduced to a single command `gulp deploy`!
 
-  The syntax to create a new Gulp task is"
+  The syntax to create a new Gulp task is:
```
Comment (markhu) [inline README.md:1080]: This is a straightforward typo fix. After this, the changes are all for MarkDown formatting, and look boring/ugly in plaintext source mode. But they do improve the readability.

File: README.md (hunk 2 above)
Comment (nelsonic) [reply README.md:607]: @markhu yeah, I have no idea what that space is. it's _pain_! (_thanks for fixing!_)

**Review by nelsonic — APPROVED** [review]
@markhu thanks for making the Markdown/Formatting fixes! hope your day is going well. ☀️

**Pull request merged**